scripts/wsrep_sst_clone.sh

```diff
@@ -0,0 +1,1065 @@
+            if [ -n "$CLIENT_SSL_CA"   ]
+            then
+                CLIENT_SSL_OPTIONS+=" --ssl-ca=$CLIENT_SSL_CA"
+                [ -n "$CLIENT_SSL_MODE" ] && CLIENT_SSL_OPTIONS+=" --ssl-mode=$CLIENT_SSL_MODE"
+            else
+                CLIENT_SSL_OPTIONS+=" --ssl-mode=REQUIRED"
+            fi
+        fi
+    else
+        wsrep_log_info "No suitable SSL configuration found. Not using SSL for SST."
+        CLEANUP_CLONE_SSL=
+        REQUIRE_SSL=
+    fi
+    # for now we force no SSL in the clone operations
+    CLEANUP_CLONE_SSL=
+    REQUIRE_SSL=
+    wsrep_log_info "SSL Not supported yet."
+}
+
+wsrep_log_debug "-> In the wsrep_sst_clone "
+
+if test -z "$WSREP_SST_OPT_HOST"; then wsrep_log_error "HOST cannot be nil"; exit $EINVAL; fi
+# MySQL client does not seem to agree to [] around IPv6 addresses
+wsrep_check_programs sed
+SST_HOST_STRIPPED=$(echo $WSREP_SST_OPT_HOST | sed 's/^\[//' | sed 's/\]$//')
+
+PROGRESS=$(parse_cnf sst progress "")
+if [ -n "$PROGRESS" -a "$PROGRESS" != "none" ]
+then
+    wsrep_log_warning "Unrecognized SST progress parameter value: '$PROGRESS'" \
+                      "Recognized values are 'none' - disable progress reporting " \
+                      "and empty string - enable progress reporting (default)" \
+                      "SST progress reporting will be disabled"
+    PROGRESS="none"
+fi
+
+# Option to CREATE USER
+WITH_OPTION="WITH caching_sha2_password"
+
+if [ "$WSREP_SST_OPT_ROLE" = "donor" ]
+then
+    echo "continue" # donor can resume updating data
+
+    if [ ${MTR_SST_DONOR_DELAY:=0} -gt 0 ]
+    then
+        wsrep_log_info "Sleeping $MTR_SST_DONOR_DELAY seconds for MTR test"
+        sleep $MTR_SST_DONOR_DELAY
+    fi
+
+    if test -z "$WSREP_SST_OPT_USER";   then wsrep_log_error "USER cannot be nil";   exit $EINVAL; fi
+    if test -z "$WSREP_SST_OPT_REMOTE_USER"; then wsrep_log_error "REMOTE_USER cannot be nil"; exit $EINVAL; fi
+    if test -z "$WSREP_SST_OPT_PORT";   then wsrep_log_error "PORT cannot be nil";   exit $EINVAL; fi
+#    if test -z "$WSREP_SST_OPT_LPORT";  then wsrep_log_error "LPORT cannot be nil";  exit $EINVAL; fi
+    if test -z "$WSREP_SST_OPT_SOCKET"; then wsrep_log_error "SOCKET cannot be nil"; exit $EINVAL; fi
+
+    CLIENT_VERSION=$($MYSQL_CLIENT --version | grep -vi MariaDB | cut -d ' ' -f 4)
+    check_client_version $CLIENT_VERSION
+
+    readonly RECIPIENT_USER=$WSREP_SST_OPT_REMOTE_USER
+    readonly RECIPIENT_PSWD=$WSREP_SST_OPT_REMOTE_PSWD
+    readonly ADMIN_USER=$WSREP_SST_OPT_USER
+    readonly ADMIN_PSWD=$WSREP_SST_OPT_PSWD
+
+    MYSQL_RCLIENT="$MYSQL_CLIENT -u$RECIPIENT_USER -h$SST_HOST_STRIPPED \
+                   -P$WSREP_SST_OPT_PORT --batch --skip_column_names --silent"
+    MYSQL_ACLIENT="$MYSQL_CLIENT -u$ADMIN_USER -S$WSREP_SST_OPT_SOCKET \
+                   --batch --skip_column_names --silent"
+
+    export MYSQL_PWD=$ADMIN_PSWD
+    if [ -z "$WSREP_SST_OPT_LPORT" ]; then
```

Comment:
Not sure about that, from original script it seems that if with MariaDB that value is filled. 
Not from config file. This because can be in auto.cnf so better to take it live

Reply:
done